error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/LexicographicalAnnotationListingTest.java

```diff
@@ -106,6 +119,13 @@ void identification() {
             "  @Bar",
             "  @Foo(ints = {1, 2})",
             "  A unsortedRepeatableAnnotation();",
+            "",
+            "  // BUG: Diagnostic matches: X",
+            "  @Baz",
```

Comment:
We could add an example that shows _only_ flagging `@{Foo,Bar}TypeUse` annotations. 
I know it is implied but might be nice to have a specific use case for it :).
```
" default @FooTypeUse @BarTypeUse A unsortedTypeAnnotations() {",
"    return null;",
"  }",
```

Reply:
Sure! Rebased and added a commit.